lib/component-repository/src/routes/virtual-components/index.js

```diff
@@ -0,0 +1,178 @@
+const express = require('express');
+const asyncHandler = require('express-async-handler');
+const bodyParser = require('body-parser').json();
```

Comment:
I would use the native functionality of express and do without the separate `bodyParser`. Basically, you only need to replace one line.
https://dev.to/taylorbeeston/you-probably-don-t-need-body-parser-in-your-express-apps-3nio

Reply:
Replaced to the native express functionality.